src/components/accordion/template.njk

```diff
@@ -0,0 +1,14 @@
+<div class="govuk-accordion" data-module="accordion">
+
+  {% for item in params.items %}
+    <div class="govuk-accordion__section {% if item.expanded %}govuk-accordion__section--expanded" {% endif %}">
+      <div class="govuk-accordion__section-header">
```

Comment:
I'm wondering if we should consider making this a `button` element and also using `aria-controls` and `aria-labelledby` to indicate the relationship? I'm just looking at a [W3C example](https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html).

Reply:
@hannalaakso I think the primary semantics here are it being a heading (`h2`), so I wouldn't want to lose that, but we could do either `<h2><button>label</button</h2>` (via javascript), or add an `aria-role="button"` attribute?

Reply:
Thanks @frankieroberto. Looking at the W3C specs, `<h2><button>label</button</h2>` is advised. I've updated the PR description.